pyproject.toml

```diff
@@ -36,25 +36,25 @@ python = "^3.7"
 pytest = ">5.4.0,<8"
 splunk-sdk = ">=1.6"
 requests = "^2.31.0"
-jsonschema = ">=4,<5"
-pytest-xdist = ">=2.3.0"
+jsonschema = "^v4.17.3"
```

Comment:
That's an interesting format, did not know we can use like this.

Reply:
Yep.